RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc

```diff
@@ -169,16 +170,16 @@ namespace PSFitter {
 
     if (step <= 0) {
       std::string txtmsg = "Parameter " + name + "  has zero or invalid step size - consider it as constant ";
-      MN_INFO_MSG2("HybridMinimizer::SetVariable", txtmsg);
+      edm::LogInfo("HybridMinimizer::SetVariable") << txtmsg << std::endl;
```

Comment:
The `std::endl` us unnecessary as MessageLogger adds the newline (unless intention is to have two newlines),
```suggestion
      edm::LogInfo("HybridMinimizer::SetVariable") << txtmsg;
```

Reply:
Actually, would it be worth to avoid the intermediate string?
```cpp
edm::LogInfo("HybridMinimizer::SetVariable") << "Parameter " << name << "  has zero or invalid step size - consider it as constant";
```
or
```cpp
edm::LogInfo("HybridMinimizer::SetVariable").format("Parameter {} has zero or invalid step size - consider it as constant", name);
```

Reply:
when the test is finished I'll add it, finished tests will keep the root rpm and avoid rebuilding it.